shared-libs/user-management/test/unit/users.spec.js

```diff
@@ -925,18 +1338,34 @@ describe('Users service', () => {
       const token = db.users.put.args[1][0].token_login.token;
       chai.expect(token.length).to.equal(64);
 
-      chai.expect(db.medic.put.args[1][0]).to.deep.nested.include({
+      const expectedDoc = {
         _id: `token:login:${token}`,
         type: 'token_login',
         reported_date: 0,
         user: 'org.couchdb.user:sally',
-        'tasks[0].messages[0].to': '+40755696969',
-        'tasks[0].messages[0].message': 'sms',
-        'tasks[0].state': 'pending',
-        'tasks[1].messages[0].to': '+40755696969',
-        'tasks[1].messages[0].message': `http://realhost/medic/login/token/${token}`,
-        'tasks[1].state': 'pending',
-      });
+        tasks: []
```

Comment:
The tasks are not getting added here any more since we are mocking the `addMessage` functionality.  I added the additional assertions for the `addMessage` call to make sure we are not loosing any fidelity in these unit tests.